indico/modules/events/abstracts/models/notification_templates.py

```diff
@@ -0,0 +1,88 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from indico.core.db import db
+from indico.util.notification_templates import NotificationTemplateBase
+
+
+def _get_next_position(context):
+    """Get the next abstracts template position for the event."""
+    event_id = context.current_parameters['event_id']
+    res = db.session.query(db.func.max(AbstractsNotificationTemplate.position)).filter_by(event_id=event_id).one()
+    return (res[0] or 0) + 1
+
+
+class AbstractsNotificationTemplate(NotificationTemplateBase, db.Model):
+    """Represents an email notification template for abstracts"""
+
+    __tablename__ = 'notification_templates'
+    __auto_table_args = ({'schema': 'event_abstracts'})
+
+    #: Whether to include authors' email addresses as To for the notification
+    #: emails
+    include_authors = db.Column(
+        db.Boolean,
+        nullable=False,
+        default=False
+    )
+    #: Whether to include submitters' email addresses as To for the
+    #: notification emails
+    include_submitters = db.Column(
+        db.Boolean,
+        nullable=False,
+        default=False
+    )
+    #: Whether to include co-authors' email addresses as CC for the
+    #: notification emails
+    include_coauthors = db.Column(
+        db.Boolean,
+        nullable=False,
+        default=False
+    )
+    #: The relative position of the template in the list of abstract
+    #: notification templates
+    position = db.Column(
```

Comment:
Hm, trying to figure out how to get the model from the context. In models all uses I can find of `context` is mainly to get the `event_id` from the `current_parameters`.